runner/src/runtime/local.rs

```diff
@@ -0,0 +1,149 @@
+use portpicker::pick_unused_port;
+use std::fs;
+use std::io;
+use std::path::{Path, PathBuf};
+use std::process::{Child, Command};
+use std::{thread, time::Duration};
+
+use super::context;
+use super::RuntimeFlavor;
+
+fn local_rpc_addr(port: u16) -> String {
+    format!("0.0.0.0:{}", port)
```

Comment:
why invalid address?

Reply:
hmm, I just used the same address in runner-js and this gets used by `near-sandbox` itself. Is it throwing invalid address for you?

Reply:
ah, didn't realize sandbox accepted this to let it auto-assign